crates/sui/src/client_commands.rs

```diff
@@ -183,6 +183,10 @@ pub enum SuiClientCommands {
         #[clap(long)]
         gas_budget: u64,
 
+        /// Optional gas price for this call
```

Comment:
Maybe a bit more context would be useful, e.g., 
```suggestion
        /// Optional gas price for this call - use only for testing and not in production enviroments.
```

Reply:
Done